source/includes/gcp/_snapshots.md

```diff
@@ -0,0 +1,146 @@
+### Snapshots
+
+Snapshots can be created to periodically back up data from your instances.
+
+#### List snapshots
+
+```shell
+curl -X GET \
+  -H 'MC-Api-key: your_api_key'
+  "https://cloudmc_endpoint/v1/services/gcp/test-area/snapshots"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "description": "This is a snapshot description",
+      "creationTimestamp": "2020-06-17T08:02:34.132-07:00",
+      "status": "READY",
+      "sourceDisk": "https://www.googleapis.com/compute/v1/projects/testarea/zones/northamerica-northeast1-a/disks/source-instance-name",
+      "sourceDiskId": "7481902077681239049",
+      "diskSizeGb": "10",
+      "storageBytes": "932828192",
+      "storageBytesStatus": "UP_TO_DATE",
+      "licenses": [
+        "https://www.googleapis.com/compute/v1/projects/debian-cloud/global/licenses/debian-9-stretch"
+      ],
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/testarea/global/snapshots/snapshot-1",
+      "labels": {
+        "sampleLabel": "label value"
+      },
+      "labelFingerprint": "6gSwJq6EkEI=",
+      "licenseCodes": [
+        "1023205"
+      ],
+      "storageLocations": [
+        "us"
+      ],
+      "kind": "compute#snapshot",
+      "downloadBytes": "932822788",
+      "id": "7059283128121212212",
+      "name": "snapshot-1"
+    },
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>
+   GET /service/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/snapshots
+</code>
+
+Retrieve a list of all snapshots.
+
+Attributes | &nbsp;
+------- | -----------
+`description` <br/>*string* | The snapshot description
```

Comment:
Just missing a period here and on description for retrieve snapshot by id

Reply:
[547f14e](https://github.com/cloudops/cloudmc-api-docs/pull/178/commits/547f14e3f03c4e0d8b89f110440893d187a778b1)